### DISCUSSION TITLE
How to speed up inference?

### DISCUSSION THREAD

**Discussion by Tareqox** [fishaudio/fish-speech#578] [Q&A]
Currently it takes around 30s-40s on a big machine (with serveral tesla v100) to generate a wav file. How can we speed this process up? On your fish audio page generation audio is much quicker.

**Comment by AnyaCoder** [comment]
Add command flag `--compile`

**Reply by Tareqox** [reply]
--compile is activated and it is still quiet slow.

**Reply by AnyaCoder** [reply]
Please refer to the speed of the second inference after compilation

**Reply by farzadsbmila** [reply]
hi @AnyaCoder 
Thanks for the hint, the speed of the second inference is pretty fast. 
Is compilation a one-time thing? ie. it happens only in the beginning, and not every time a new prompt is given?
